notebooks/official/prediction/custom_batch_prediction_feature_filter.ipynb

```diff
@@ -0,0 +1,1586 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "mxIxvDdglugx"
+      },
+      "outputs": [],
+      "source": [
+        "job = aiplatform.CustomTrainingJob(\n",
+        "    display_name=JOB_NAME,\n",
+        "    script_path=\"task.py\",\n",
+        "    container_uri=TRAIN_IMAGE,\n",
+        "    requirements=[\"google-cloud-bigquery>=2.20.0\", \"db-dtypes\"],\n",
+        "    model_serving_container_image_uri=DEPLOY_IMAGE,\n",
+        ")\n",
+        "\n",
+        "MODEL_DISPLAY_NAME = \"penguins-unique\"\n",
+        "\n",
+        "# Start the training\n",
+        "if TRAIN_GPU:\n",
+        "    model = job.run(\n",
+        "        dataset=dataset,\n",
+        "        model_display_name=MODEL_DISPLAY_NAME,\n",
+        "        bigquery_destination=f\"bq://{PROJECT_ID}\",\n",
+        "        args=CMDARGS,\n",
+        "        replica_count=1,\n",
+        "        machine_type=TRAIN_COMPUTE,\n",
+        "        accelerator_type=TRAIN_GPU.name,\n",
+        "        accelerator_count=TRAIN_NGPU,\n",
+        "    )\n",
+        "else:\n",
+        "    model = job.run(\n",
+        "        dataset=dataset,\n",
+        "        model_display_name=MODEL_DISPLAY_NAME,\n",
+        "        bigquery_destination=f\"bq://{PROJECT_ID}\",\n",
+        "        args=CMDARGS,\n",
+        "        replica_count=1,\n",
+        "        machine_type=TRAIN_COMPUTE,\n",
+        "        accelerator_count=0,\n",
+        "    )"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "make_prediction"
+      },
+      "source": [
+        "## Send Batch Prediction job request with filtered features (instanceConfig field)\n",
+        "\n",
+        "Now that the model is ready, you can send batch prediction request directly from the model resource without needing to deploy the model to an endpoint.  For data types that support both batch and online predictions, use batch predictions when you don't require an immediate response and want to process accumulated data by using a single request. \n",
+        "\n",
+        "There are many times that the input data you have for batch prediction need some preprocessing before the predictor container can actually take the request. The most common preprocessing heard are feature filtering and instance container transforming. The feature filtering removes instance identifiers or input metadata that is stored in the dataset for various reasons but not for the predictors. You can filter and/or transform your batch input by specifying `instanceConfig` in your `BatchPredictionJob` request (**v1beta1 only**).\n",
+        "\n",
+        "In this notebook you learn how to send batch prediction request by including or excluding a list of features in the request using `instanceConfig` field so the prediction job gets the right set of features regardless of the original input feature set.\n",
```

Comment:
consider linking to instanceConfig https://cloud.google.com/vertex-ai/docs/reference/rest/v1beta1/projects.locations.batchPredictionJobs#instanceconfig

Reply:
Done.